resource/subfunctions/started_task.md

```diff
@@ -0,0 +1,276 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Subfunctions
+---
+
+# Started task
+
+Resource Administration functions for Started Task Administration
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .note }
+> _This may not be the only step to manage a **Started Task** in your environment._
+> _You may also have to refresh the **STARTED** class to enact these changes._
+> _Please consult RACF documentation and manuals for an understanding of the **STARTED** class._
+
+## `ResourceAdmin.add_started_task()`
+
+```python
+  def add_started_task( self, started_task_name: str, traits: dict = {} ) -> Union[dict, bytes]:
+```

```

Comment:
```python
def add_started_task(self, started_task_name: str, traits: dict = {}) -> Union[dict, bytes]:
```

Reply:
Remove leading spaces.

Reply:
Leading spaces and unnecessary trailing spaces and indents removed across all function definitions.